### PR TITLE
Migrate from tslint to eslint + @typescript-eslint

### LINKED ISSUE #2: Migrate to ESLint instead of TSLint
As per https://github.com/palantir/tslint/issues/4534, we need to migrate TSLint to ESLint.

### PR DESCRIPTION
**Pull request by Namchee**
This pull request handles migration from the deprecated `tslint` to `eslint` and `@typescript-eslint`. Will close #2.

NB: I don't know your personal rules and coding styles, so you'll probably have to do a little bit modification on `.eslint.config.json`

### CODE REVIEW

**Comment by chez14** [comment]
This looks good, imo, gonna do further check after work. Thank you for the PR!

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    "warn",
+                    "info"
+                ]
+            }
+        ],
+        "curly": [
+            "error",
+            "all"
+        ],
+        "lines-between-class-members": [
```
Comment (chez14) [inline .eslintrc.json:48]: Is there any way to configure this to 3 lines for each class members?

File: package.json [hunk 2]
```diff
@@ -30,8 +31,8 @@
   "scripts": {
     "test": "jest",
     "build": "lerna run build",
-    "lint:format": "tslint -p tsconfig.json --fix",
-    "lint": "tslint -p tsconfig.json",
+    "lint:format": "eslint --fix packages/ --ext .js,.ts",
```
Comment (chez14) [inline package.json:34]: Is there any way to set it finely? I mean, I need to exclude the `build` path for each project. I usually test by build the lib before publishing, to check if they can be used in non TS environment.

**Review by chez14 — CHANGES_REQUESTED** [review]
Hi @Namchee,

After further check, there are some questions and probably change request.

Also, can you remove the `package.lock.json` and add them to `.gitignore`. I use Yarn and it complained complained:
> package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.

Thank you!

File: .eslintrc.json (hunk 1 above)
Comment (Namchee) [reply .eslintrc.json:48]: According to [this](https://eslint.org/docs/rules/lines-between-class-members), no.

Comment (chez14) [reply .eslintrc.json:48]: x.x aww... that's so unfortunate.

File: package.json (hunk 2 above)
Comment (Namchee) [reply package.json:34]: You can use [.eslintignore](https://eslint.org/docs/user-guide/configuring#eslintignore)

Comment (chez14) [reply package.json:34]: Got it. I think I'll configure that by my self, thank you!

**Review by chez14 — APPROVED** [review]

**Comment by chez14** [comment]
As I wanted to configure some behaviour of the linter, I'll add the skip-ci to prevent the CI to run the automated publishing.

Once again, thank you for the contribution! :D

**Comment by repurika** [comment]
:rocket: PR was released in [`v1.0.1`](https://github.com/chez14/node-mal-api/releases/tag/v1.0.1) :rocket:

**Pull request merged**